lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -406,11 +406,11 @@ WasmBinaryReader::EndOfModule()
 void
 WasmBinaryReader::ReadMemorySection()
 {
-    // TODO: change to use multiple of page size
-    uint32 minSize = 1 << ReadConst<UINT8>();
-    uint32 maxSize = 1 << ReadConst<UINT8>();
+    UINT length = 0;
+    UINT minPage = LEB128(length);
+    UINT maxPage = LEB128(length);
```

Comment:
Currently it seems that we aren't doing real validation that section length == declared length, so doesn't really matter. But we will need to add in. Functionally it's fine, because the callee updates modulestate based on the declared section length.